src/components/ha-date-range-picker.ts

```diff
@@ -71,7 +71,11 @@ export class HaDateRangePicker extends LitElement {
       this._openingDirection = "center";
     }
 
-    if (!this.hasUpdated && this.ranges === undefined) {
+    if (
+      (this.hasUpdated && this.ranges === undefined) ||
```

Comment:
Why was this changed, is that an accident?
```suggestion
      (!this.hasUpdated && this.ranges === undefined) ||
```

Reply:
huh, good spot. I didn't intend to do that, not sure how that happened.